core/src/test/java/com/yahoo/gondola/core/GondolaTest.java

```diff
@@ -105,6 +108,16 @@ public void doAfterMethod(ITestContext tc, ITestResult tr, Method m) throws Exce
         gondolaRc.stop();
     }
 
+    @AfterTest(alwaysRun = true)
+    public void doAfterTest() {
+        // Print timings
```

Comment:
maven plugin `surefire` already integrate with testng and generate a report here: ` target/surefire-reports/testng-results.xml` , guess this is what you need. publish the report on TravisCI and track that.